baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StreamOfEmpty.java

```diff
@@ -33,7 +34,7 @@
         name = "StreamOfEmpty",
         link = "https://github.com/palantir/gradle-baseline#baseline-error-prone-checks",
         linkType = BugPattern.LinkType.CUSTOM,
-        severity = BugPattern.SeverityLevel.SUGGESTION,
+        severity = SeverityLevel.WARNING,
```

Comment:
same here, definitely a bug

Reply:
I think the risk with making some of these errors is that repos which have a bunch of existing violation will be blocked. `WARNING` seems a bit safer because it's won't fail compilation unless you have `-Werror`.

Happy to change it if you would still prefer `ERROR`, just wanted to point this out.

Reply:
in practice these rules will be auto applied on upgrade so I'm not particularly worried about blocking upgrades.